graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -51,17 +51,17 @@ def _validate_edges_do_not_have_extra_links(class_name, properties):
             continue
 
         if property_descriptor.type_id == PROPERTY_TYPE_LINK_ID:
-            raise IllegalSchemaError(u'Edge class "{}" has a property of type Link that is not '
-                                     u'an edge endpoint, this is not allowed: '
-                                     u'{}'.format(class_name, property_name))
+            raise IllegalSchemaStateError(u'Edge class "{}" has a property of type Link that is'
+                                          u' not an edge endpoint, this is not allowed: '
```

Comment:
By our adopted convention, spaces go at the ends of lines -- we don't start string literals with a space.